### ISSUE TITLE
RPA does not run on windows task scheduler - some suggestions for scheduling

### ISSUE THREAD

**Issue by chlimaferreira** [kelaberetiv/TagUI#900]
Ken Soh, is there any way to run the RPA library on the Windows scheduler? 

 I am teaching the rpa library to my students, but I was in no way able to schedule execution without third party software.  I would also replace my Selenium scripts with RPA.

Follow the course link a ru teaching your rpa library:  https://www.udemy.com/course/curso-robotic-process-automation-rpa-crie-robos-bots-com-python/?couponCode=RPACOMPYTHON

**Comment by kensoh** [comment]
Hi Charles, yes, from Windows task scheduler you should be able to schedule new tasks to run at your desired time.

For TagUI, you can do it in 2 ways. 1st way is provide full path to run tagui\src\tagui.cmd and then the full path to the flow to run. 2nd way is your run tagui command with -deploy option first. It will generate a deploy batch file for you. Then in task scheduler you can just call the deploy batch file.

For RPA for Python, you can test the python command to run your script, then schedule task to do the same thing. For eg `python3 automation.py`. Important thing is make sure that packages are installed to the right Python version, so that when you call Python, it is able to find the package to run.

Let me know if you run into any problems doing any of the above suggestions!

**Comment by chlimaferreira** [comment]
hello, @Keh Soh!  I will test and return here.

**Comment by chlimaferreira** [comment]
hi @Ken Soh, I did the tests and look at the outputs, I was not successful.
The python3 automation command works at the prompt, but in the windows task scheduler it didn't work.
![Erros de Execução](https://user-images.githubusercontent.com/28329092/103419555-4ae7eb00-4b72-11eb-91e1-d1651506dade.PNG)

**Comment by kensoh** [comment]
Hi Charles, above screenshot shows that you are running tagui command on your Python script. The tagui command is only used for running TagUI scripts. If you are running some Python scripts you should be running with python2, python3 or python command. For Task Scheduler with Python scripts, I'm not familiar but below are examples -

https://www.jcchouinard.com/python-automation-using-task-scheduler/
https://datatofish.com/python-script-windows-scheduler/
https://towardsdatascience.com/automate-your-python-scripts-with-task-scheduler-661d0a40b279

**Comment by chlimaferreira** [comment]
Hi Keh, I managed to solve it as follows:
I created another venv, in the task scheduler, in Program I call python.exe from within this new venv, in arguments I call the automation path.

For some reason a problem is generated in the venv in which the rpa library (tagui) is installed, preventing the manipulation of files by the windows task scheduler. I hope to help colleagues who have had the same problem.
Shares on Github / RPA-Python too.
